src/grandorgue/sound/GOSoundToneBalance.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2006 Milan Digital Audio LLC
+ * Copyright 2009-2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+
+#ifndef GOSOUNDTONEBALANCE_H_
+#define GOSOUNDTONEBALANCE_H_
+
+#include "GOSoundFilter.h"
+#include <cmath>
+
+class GOSoundToneBalance {
+public:
+  void Init(int slider) {
```

Comment:
This code is quite long and it is not needed to be inline. So move it to the `.cpp` file, please